rhel/SPECS/pmm-server.spec

```diff
@@ -59,8 +101,18 @@ rm -rf %{pmm_repo}-%{pmm_commit}
 %{_sysconfdir}/alertmanager.yml
 %{_datadir}/%{name}
 
+%files systemd
+%license LICENSE
+%doc README.md CHANGELOG.md
+%{_sysconfdir}/%{repo}
+%config(noreplace) %{_sysconfdir}/%{repo}/pmm-server.env
+%{_unitdir}/pmm-server.service
 
 %changelog
+
+* Tue May 31 2022 Denys Kondratenko <denys.kondratenko@percona.com> - 2.29.0-2
+- PMM-7925 add systemd service with podman support
+
 * Mon May 16 2022 Nikita Beletskii <nikita.beletskii@percona.com> - 2.29.0-1
 - PMM-10027 remove useless packages
 
```

Comment:
Why do we use this spec? We use this spec for internal pmm-server package (it contains some js files) and I'm not sure that we need systemd service in container :)

Reply:
it was easier for PoC to use existing spec. And actually it fits it well in term that is a part of pmm-server. We don't need it in container for sure, but probably after build we need to sync it into some repo that available for everyone.